cpp/ql/src/semmle/code/cpp/Function.qll

```diff
@@ -393,18 +393,23 @@ class Function extends Declaration, ControlFlowNode, AccessHolder, @function {
 
   /** Gets a function that overloads this one. */
   Function getAnOverload() {
```

Comment:
Given https://github.com/github/codeql/issues/4889#issuecomment-752658377, I suggest you add a remark to this QLDoc to say that if _overrides_ are wanted rather than _overloads_ then `MemberFunction::getAnOverridingFunction` should be used instead.

Reply:
That's a good idea! I've added such a remark in 134982c5a9c37ca0d5f86463fe33b3dab8bae970.